src/main/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecute.java

```diff
@@ -229,9 +231,9 @@ public static final class Config extends AbstractBigQueryActionConfig {
     @Name(NAME_CMEK_KEY)
     @Macro
     @Nullable
-    @Description("The GCP customer managed encryption key (CMEK) name used to encrypt data written to " +
-      "any dataset or table created by the plugin. If the dataset or table already exists, this is ignored. More " +
-      "information can be found at https://cloud.google.com/data-fusion/docs/how-to/customer-managed-encryption-keys")
+    @Description("The GCP customer managed encryption key (CMEK) name used to encrypt data written to the " +
```

Comment:
"The GCP customer managed encryption key (CMEK) name used to encrypt the query results. It's only applicable when users choose to store the query results in a BigQuery table.  More information can be found at " +
      "https://cloud.google.com/data-fusion/docs/how-to/customer-managed-encryption-keys"